### PR TITLE
Support for nb-NO

### PR DESCRIPTION
**Pull request by Goddesen**
Added support for Norwegian ingame language, as well as currency (kr).

`nb-NO.txt` is already merged in OpenRCT2/Localisation

### CODE REVIEW

**Comment by LinusU** [comment]
The indentation looks off on github, I think it's a tab vs spaces issues. Could you update to match the rest of the code?

**Comment by Gymnasiast** [comment]
I'll get back to this soon.

File: src/localisation/language.cpp [hunk 1]
```diff
@@ -41,6 +41,7 @@ enum {
 	RCT2_LANGUAGE_ID_CHINESE_TRADITIONAL,
 	RCT2_LANGUAGE_ID_12,
 	RCT2_LANGUAGE_ID_PORTUGUESE,
+    RCT2_LANGUAGE_ID_NORWEGIAN,
```
Comment (AaronVanGeffen) [inline src/localisation/language.cpp]: This is a mapping for the languages in the original game. You probably shouldn't modify this enum, and use English UK below instead.

**Comment by Goddesen** [comment]
@LinusU I see. The guidelines say to indent with 4 spaces, I figured that was fine. I'll change it to match if that's what is best for now?

**Comment by IntelOrca** [comment]
@Goddesen that's only for new C++ files, sorry for the confusion.

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -4121,6 +4121,7 @@ STR_5811    :{SMALLFONT}{BLACK}If checked, you can have up to{NEWLINE}255 cars p
 STR_5812    :Show multiplayer window
 STR_5813    :{OPENQUOTES}{STRING}{ENDQUOTES}
 STR_5814    :{WINDOW_COLOUR_1}{OPENQUOTES}{STRING}{ENDQUOTES}
+STR_5871    :Krone (kr)
```
Comment (Wirlie) [inline data/language/en-GB.txt]: I think that this should be moved until after STR_5870, to maintain order (and prevent confusion).

Comment (Goddesen) [inline data/language/en-GB.txt]: There's a hole section labeled #tooltips, I figured it might be confusing to put it right there. But I could do that.

Comment (Wirlie) [inline data/language/en-GB.txt]: You're right, but that section refers to STR_5815 - STR_5859.
PS: Good point, it is appropriate to add a space or comment after this section.

**Comment by Gymnasiast** [comment]
Don't duplicate krone. Put a space before it please, as I said in Localisation.

**Comment by Goddesen** [comment]
I dun goofed.

**Comment by Nubbie** [comment]
oh no please

**Pull request merged**